client/cli/go/cmds/volume.go

```diff
@@ -139,21 +146,24 @@ var volumeCreateCommand = &cobra.Command{
 			return fmt.Errorf("Missing endpoint")
 		}
 
-		// Check clusters
-		var clusters_ []string
-		if clusters != "" {
-			clusters_ = strings.Split(clusters, ",")
-		}
-
```

Comment:
I still would have liked to see this cleanup as part of a preliminary / preparatory patch, because it is a code cleanup and not directly related to the contents of this commit.